packages/notifications/push/amplify_push_notifications_pinpoint/pubspec.yaml

```diff
@@ -22,9 +22,8 @@ dependencies:
 
 dev_dependencies:
   amplify_lints: ^2.0.0
-  amplify_test:
-    path: ../../../test/amplify_test
-  build_runner: ^2.0.0
+  amplify_test: any
```

Comment:
Must use a path dependency for this. See https://github.com/aws-amplify/amplify-flutter/pull/2865